scripts/commands/checkLinks.ts

```diff
@@ -23,8 +23,8 @@ import rehypeRemark from "rehype-remark";
 import rehypeParse from "rehype-parse";
 import remarkGfm from "remark-gfm";
 
-// These files are not searched to see if their own links are valid.
-const IGNORED_FILES: string[] = [];
+// The links in a file are not searched to see if they are valid.
+const IGNORED_FILES: { [id: string]: string[] } = {};
```

Comment:
What about calling it `FILES_TO_IGNORES`? It should make it easier to understand call sites of this variable.

Reply:
You are right, thanks!